### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by erkarl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
+# [1.1.0](https://github.com/ExchangeUnion/xud/compare/v1.0.0...v1.1.0) (2020-09-28)
+
+Most notable features of this release:
+- Xud now prevents orders from entering the orderbook if there's insufficient inbound liquidity for Connext currencies (ETH/ERC20). Inbound liquidity is automatically requested from the Connext node. This smart collateralization feature considerably increases the likelihood that the orders can be executed.
```
Comment (kilrau) [inline CHANGELOG.md:4]:
```suggestion
- Xud now automatically requests inbound liquidity for connext currencies (ETH/ERC20) and prevents orders from entering the orderbook until there is sufficient inbound liquidity to cover the order. This "smart collateralization" feature considerably increases the likelihood that the orders can be executed successfully.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+- [DAI](https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f) token and `USDT/DAI` trading pair has been enabled by default.
```
Comment (kilrau) [inline CHANGELOG.md:5]:
```suggestion
- The [DAI](https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f) token and `USDT/DAI` trading pair have been enabled by default.
```

**Review by kilrau — APPROVED** [review]
Small nits only

**Review by kilrau — APPROVED** [review]

**Review by sangaman — APPROVED** [review]
FWIW, for the 1.0.0 release I put my written notes just in the github release message, not in the changelog.md, but it makes sense to do it this way too. I may go and retroactively add my 1.0.0 notes to the repo.

**Pull request merged**